### PR TITLE
Feature/styled components

### PR DESCRIPTION
**Pull request by henkegithub**
styled app

### CODE REVIEW

**Review by DorisW3 — COMMENTED** [review]
Hey Michael, 
your app looks great! I like it very much. There are only little things I would change.
And I have one question: Why do you use css styling additional to styled components?
Sorry for the delay!

**Comment by henkegithub** [comment]
> Screenshot 2023-08-28 at 16 44 16 I get this error anyone knows how to fix it?

is fixed, was a p in a p

**Comment by henkegithub** [comment]
> Hey Michael,
> your app looks great! I like it very much. There are only little things I would change.
> And I have one question: Why do you use css styling additional to styled components?
> Sorry for the delay!

@DorisW3 I have no idea what you mean by css styling additional to styled components

File: components/Product/index.js [hunk 1]
```diff
@@ -15,19 +21,23 @@ export function Product({ id, name, price, handleAmountChange, amount = 0 }) {
...
-      <button className="reset" onClick={reset}>
-        Reset
-      </button>
-    </>
+    <StyledList>
+      <StyledMenuItem>
+        <StyledParagraph>{name}</StyledParagraph>
+        <StyledParagraphPrice>{price} €</StyledParagraphPrice>
+        <StyledDiv>
+          <StyledOrderButton className="control__btn" onClick={increase}>
```
Comment (DorisW3) [inline components/Product/index.js:29]: hi michael, I meant the className you use in line29, 33 and 36. :-)

**Pull request merged**